apps/smartling/lambda/src/index.ts

```diff
@@ -148,6 +148,9 @@ export function makeApp(fetchFn: any, issuer: any) {
       '/frontend',
       express.static(path.dirname(require.resolve('@contentful/smartling-frontend')), {
         lastModified,
+        setHeaders: (res: any) => {
```

Comment:
lastModified was passed in as a boolean option but this didn't ultimately set the lastModified date in the header, causing an issue with serving up the assets on the frontend. This change explicitly sets the last modified header with the last modified date as a fix.